docs/understanding-tracking-design/create-tracking-plan-with-tracking-scenarios/managing-tracking-scenarios-console/index.md

```diff
@@ -0,0 +1,166 @@
+---
+title: "Managing tracking scenarios in the console"
+sidebar_label: "🆕 Managing tracking scenarios in the console"
+sidebar_position: 80
+---
+
+
+## Creating a New Tracking Scenario using Console
+
+Tracking Scenarios are directly linked to Data Structures and represent the events that are documented and tracked using a certain Data Structure.
+
+To create a new Tracking Scenario, navigate to the "Data Structures" option in the menu. From there, select a data structure from the list.
+
+:::info
+Note that Tracking Scenarios can only be created from Data Structures that have been set as "Event."
+:::
+
+After selecting a data structure, click on the "Tracking Scenarios" tab, as shown in the image.
+
+![](images/ts-1.png)
+
+Next, click the 'Add new scenario' button. A new dialog box will appear with fields for basic information that can be defined during creation, as shown.
+
+![](images/ts-2.png)
+
+Tracking scenarios should be created in association with the most recent version of the data structure deployed in production. This will determine the rules for tracking implementation that can be set in the validation part of the tracking scenarios.
+
+It is always guaranteed that your tracking scenario will have rules compatible with the associated data structure version. This ensures that any event tracked by following the rules defined in the tracking scenarios will pass validation for the associated data structure.
+
+However, if a newer version of the data structure is subsequently deployed in production, you may wish to "upgrade" the tracking scenario and tracking to the newer data structure and verify that your implementation rules still meet validation.
+
+The fields available for filling at this stage are:
+
+- A text field for the **scenario name**
+- A tag field to indicate which applications this scenario is designed to track.
+- A free text field for a **description** of the tracking scenario.
+
+![](images/ts-3.png)
```

Comment:
This needs a new screenshot as the system is not searching for existing tags whilst typing.
I think there is also a Jira ticket to fix the UI.

Reply:
How about this one?
![Screenshot 2023-06-26 at 09 31 54](https://github.com/snowplow/documentation/assets/6108743/d025bbee-6a86-43a1-b88f-4aa34d2468ee)

Reply:
Sounds good!